### ISSUE TITLE
Ask if there is a fee to use an ATM

### ISSUE THREAD

**Issue by arrival-spring** [streetcomplete/StreetComplete#3100]
I think it's helpful to know whether or not an ATM charges you to withdraw cash, so it would be good to add `fee=yes` or `fee=no` where neither `fee` nor `charge` is already tagged.

Maybe this could be a new quest or maybe it could just be added to the `general_fee` quest (although the other things in that quest are quite different to an ATM).

This is listed on the wiki as a combination, but isn't hugely widely used, ~0.5% have it tagged according to taginfo.

The wiki suggests an option of "some" if not everyone needs to pay, but this is only used 356 times in all uses of `fee=*` so I think that's rare enough that it would be covered by adding a note.

**Comment by matkoniecz** [comment]
https://github.com/streetcomplete/StreetComplete/issues/new/choose has new quest template: https://github.com/streetcomplete/StreetComplete/issues/new?assignees=&labels=&template=quest-suggestion.md

With link to https://github.com/streetcomplete/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete and template that includes

> Easily answerable by everyone from the outside but a survey is necessary

where and how can I answer it? In Poland it depends on bank, account type, transaction type and other factors and is unsigned.

**Comment by arrival-spring** [comment]
Maybe it could be only enabled in certain countries then. In the UK ATMs legally have to clearly state whether or not there is a charge (before you even put your card in), and either there is a charge for everyone or for noone.

**Comment by matkoniecz** [comment]
In such places it makes sense (a good law BTW).

Can you link or upload some photo? Is it appearing on the screen? 

I looked at some https://commons.wikimedia.org/wiki/Category:Automatic_teller_machines_in_London and I do not see it (but often quality of photos is low)

https://commons.wikimedia.org/wiki/File:Gold_ATM,_Barclays,_Enfield.jpg
https://commons.wikimedia.org/wiki/File:Cardtronics_CashZone_ATM,_Green_Lanes,_London.JPG

**Comment by arrival-spring** [comment]
On this one you linked, if you look closely you can see it says on the screen "This machine charges £??? for cash withdrawals.
https://upload.wikimedia.org/wikipedia/commons/6/62/Cardtronics_CashZone_ATM%2C_Green_Lanes%2C_London.JPG

The other one is a high street bank and they rarely if ever charge for cash withdrawals. I expect when you went to use it it would say something like "This machine does not charge for cash withdrawals".

On this one it clearly says above it "Free cash withdrawals"
https://commons.wikimedia.org/wiki/Category:Automatic_teller_machines_in_London#/media/File:Kings_Cross_Post_Office.jpg

This is a good image of what the screen looks like when it charges you https://upload.wikimedia.org/wikipedia/commons/7/78/Street_ATM_north_Finchley.JPG

Generally if it's free it says it above and will probably also say it on the screen, maybe always say it on the screen. If there's a fee it will usually only say that on the screen (I guess so they can change the fee just with software).

**Comment by smichel17** [comment]
I am not 100% sure of the information below. It is my **_impression_ as an everyday person** of how things work in the US:

- Nearly all ATMs would be `fee=some`.
- Some ATMs have a signed fee. I cannot recall any ATMs where I specifically noticed they were unsigned, but I can recall ATMs where I do not remember seeing a sign, so I suspect signage is not mandatory.
- In addition to the name of the bank, some ATMs feature a *network* name (or, more likely, icons). In this case, I'd expect in-network banks would have no fee and other banks would have a fee.

I have no point here, just sharing info. Ignore it if it's not helpful.

**Comment by westnordost** [comment]
Yes, I'll close this as will not fix. 

My experience everywhere is that if you pay a fee or not always depends on which bank network the bank you got the card from is in and in which bank network the atm is in. I've never seen any ATM that does not charge any fee at all unconditionally. `operator` and/or `network` on ATMs is much more useful and this is already collected with this app.

**Comment by matkoniecz** [comment]
> On this one it clearly says above it "Free cash withdrawals"
> https://commons.wikimedia.org/wiki/Category:Automatic_teller_machines_in_London#/media/File:Kings_Cross_Post_Office.jpg

Are there many actually unconditionally free ATMs? Also for - for example - foreigners? Are they guaranteed to be free also from charges by bank operating account?

**Comment by arrival-spring** [comment]
If they're "Free" then that means you're not charge a fee by that ATM to get cash out from there. If your bank account charges you to get cash out at any ATM or if there are any foreign transaction fees then that would still apply, but I would definitely still tag something like that as `fee=no`.

**Comment by mnalis** [comment]
@arrival-spring I would think exactly the opposite... If I put a debit card in ATM and it gives me banknote of 50 EUR (or whatever), if the balance on my bank account changes by anything more than 50.00 EUR, then it is by definition **not free**, eg. has **some fee** applied. **Who exactly took which part of that fee, is much less relevant**  to me as a customer of ATM.

In my experience (traveling around mostly Europe, but with some visits to Canada/USA/Australia), it is almost never `fee=no` when taking cash from [ATM](https://en.wikipedia.org/wiki/Automated_teller_machine), while it is almost always `fee=no` when paying for goods/services via [EFTPOS](https://en.wikipedia.org/wiki/EFTPOS) in shops (unless there is currency conversion involved, but that is taken for granted). Basically, only time I've ever seen the ATM **not** charge **some** fee (by definition of _fee_ specified above) is if **all of the following are true**:
- you are using debit card (and not credit card) to withdraw cash at ATM
- you are using it in the same country of origin (eg. bank account / card provider and ATM are in same country)
- ATM is owned by same bank as your debit card bank / account.

Theoretically, you might get without a fee if using same bank group which span multiple countries; but usually  it just results in somewhat lesser fee then the other ATMs.

Otherwise, they're very inventive, there can be various fees (and often are several of them):
- charge by ATM company
- charge by your bank
- charge by card provider (not necessarily the same corporate entity as you bank)
- charge by international intermediaries (ie. two small banks do not interconnect directly, but over a bigger bank)
- unfavorable conversion rates in not the same currency of card/bank account and cash (sometimes several of them! The longest chain I've seen was I think HRK=>EUR=>USD=>UAH for single transaction)

And each of those charges can be (combination of):
- fixed amount per withdrawal
- percentage of the amount being withdrawn (often with minimum and sometimes with maximum fee limits)
- variable depending on the banks / ATM operators / card companies as well as countries involved
- some various charges depending on the amount being above/below some arbitrary amount (or less arbitrary, eg. depending on the amount of funds available, like for example overdraw attempts)

In other words, I find it highly unlikely that the ATM could even _display on such small screen all the rules by which your money is going to be taken from you_ (eg. "_the fee_") before you put your card in. Now, if it was offering 60-page PDF, I'd might find it more believable. Most (all?) of them are unable (or unwilling, but I'll give them a benefit of the doubt) to display the _total fee_ **even after** you put your card in _and_  enter exact amount of money you want to withdraw.

Most likely IMHO, it that the ATM is required by law (and thus displaying) **just one small part** of the _total fee_ that is going to be taken from unsuspecting user.

**Comment by arrival-spring** [comment]
Hmm, it sounds like I hadn't appreciated how different the situation is in other places.

In reference to this:

> Basically, only time I've ever seen the ATM **not** charge **some** fee (by definition of _fee_ specified above) is if **all of the following are true**:
> 
> * you are using debit card (and not credit card) to withdraw cash at ATM
> * you are using it in the same country of origin (eg. bank account / card provider and ATM are in same country)
> * ATM is owned by same bank as your debit card bank / account.

Here it's just the first two that need to apply for there to be no fee (if the cash machine says it is free).
So that means that for >99% of withdrawals there is no fee.

There are quite a number of ATMs which aren't owned by a bank, and often it is those that charge a fee, and whilst it would probably be possible to derive whether or not there's a fee based on the operator it would be best to be tagged specifically.

i.e. I have a debit card and I want to get cash out without being charged, I would want to search for `amenity=atm` and `fee=no`.

**Comment by mnalis** [comment]
> Here it's just the first two that need to apply for there to be no fee (if the cash machine says it is free).
So that means that for >99% of withdrawals there is no fee.

@arrival-spring  Only if your country has <1% travelers/tourist (which I find hard to believe, unless your country is active warzone). But I might be biased, as my Croatia with ~4 million residents gets more than 16 million visitors per year (at least it used to pre-COVID19). That is about 400% more tourists than residents every year. So even in each and every ATM here charged fee only for foreigners (which is not the case), `fee=no` would be simply wrong in more than 75% of the cases.

Sure, Croatia is the outlier here, but even if your country gets only 25% tourists annually (UK for example has about 58% [international visitors](https://en.wikipedia.org/wiki/World_Tourism_rankings#Europe), or Germany 47%, etc.), `fee=no` is still a very wrong **misinformation that will affect huge number of people** (especially as those not familiar with the town and thus the one mostly in need of this information are tourist). And next year there is another batch of tourists (many of whom are not the same as year before), so ratio only gets worse with each passing year!

As indicated before, SC does mark `operator` on ATMs, and a resident is more likely to know which operators are "safe" (eg. no-fee) in his town/country (as opposed to a tourist which would have no idea), so if anything, default case should be tailored toward tourists (eg. if a tourist must pay but locals do not, and you want to mark `fee`, mark it with `fee=yes` + optionally `fee:conditional=*` if you know the details, or `fee=some` if you don't)

So, if you do decide to mark those ATM which **do not have fees for all domestic users only**, please consider marking them with `fee=international` or `fee=some` or something [else](https://wiki.openstreetmap.org/wiki/Conditional_restrictions) (like `fee=yes` + `fee:conditional=no @ national` ?) instead of **_incorrect_** `fee=no`.

Consider those situations:
- If I were to find `amenity=toilet` with `fee=no`, I would assume I (or anybody else) can use it without paying. If it is for example inside a cafe, and you can use it **"for free" only** if you're paying customer (and must pay to get access otherwise), then I'd expect `fee=non_customers` or something similar (like [actually documented](https://wiki.openstreetmap.org/wiki/Key:fee) `fee=yes` + `fee:conditional=no @ customer` combination), instead of (quite wrong) `fee=no`.

- Or if there was `amenity=parking` with `fee=no`, I'd assume **anybody** can park there without paying, as opposed to parking being free only for permit holders or residents of that neighborhood, and charged for everybody else. Let's not add incorrect data just because it may _might be correct in some cases_.

**Comment by arrival-spring** [comment]
Ok, so how should the difference between the ATMs be tagged?
One that charges you £2 to withdraw cash and one that doesn't (as a resident of the country using a debit card).

**Comment by matkoniecz** [comment]
That would be better asked on tagging mailing list ( https://wiki.openstreetmap.org/wiki/Tagging_mailing_list ), maybe https://lists.openstreetmap.org/listinfo/talk-gb maybe on other of contact channels ( https://wiki.openstreetmap.org/wiki/Contact_channels ).

Especially if there is no established tagging scheme discussing new one should not take place on random issue in bug tracker of one editor.

**Comment by mnalis** [comment]
@arrival-spring I've given few possibilities (eg. `fee=yes` for one that charges everyone, and `fee=some` for one that only charges sometimes; or `fee:conditional` schema). 

There is also always the possibility to use (standalone, or in addition to above) good old free-form [key:description](https://wiki.openstreetmap.org/wiki/Key:description)  where you can explain the exact payment requirements (best of all, that one is actually quite often supported by end-user apps, like OsmAnd).

**Comment by HolgerJeromin** [comment]
You/mappers will always loose in this topic: 
https://www.ing.de/girokonto/karten-bargeld/
1) Getting cash from the VISA from this bank will provide you with free :-) money nearly everywhere in Europe.
2) Getting cash from the girocard from this bank will provide you with free :-) money only with the few ING ATMs. With all other ATMs there will be a fee.

So every ATMs in Europe are `fee=no` because of 1 or are all non ING ATMs in Germany `fee=yes` because of 2?

**Comment by egelmex** [comment]
In the UK both free and charged ATMs exist. Our ATMs are all part of the Link system. https://en.wikipedia.org/wiki/LINK_(UK). Some charge some don't but it doesn't matter which bank you are with. 

> As of 2016, there were about 54,000 free to use cash machines, of which 23,600 were provided by independent suppliers, and 16,000 cash machines that charge for withdrawals. 
-- Wikipedia

In the UK knowing if a an ATM is free or not is very useful.

**Comment by mnalis** [comment]
> In the UK both free and charged ATMs exist. Our ATMs are all part of the Link system. https://en.wikipedia.org/wiki/LINK_(UK). Some charge some don't but it doesn't matter which bank you are with.

... as long as that bank is LINK member, I presume?

But if I come with some foreign card (eg. not issued by one of that 38 LINK members from UK), would I be charged a fee or not (on those ATMs claiming to be without fee)? The wiki page is not detailed, but it seems to imply that only LINK members (which covers most of UK issued cards) settle the interchange fees between themselves - so would they be charging issuers which are not members of LINK (like cards issued in most of rest of the world?)

**Comment by egelmex** [comment]
My understanding is that out of network fees are not managed directly by the UK ATMs, a $1-3 charge may be passed on by the card holders bank, but this is on the card holders bank side to decide not the ATM.

**Comment by mnalis** [comment]
@egelmex You should read whole discussion if you're interested in the subject, especially for example https://github.com/streetcomplete/StreetComplete/issues/3100#issuecomment-889932452. 

If `amenity=parking` with `access=yes` was marked with `fee=no` and an **international user was charged a fee** because _"well, it is free for domestic users only, and the fee is actually not charged by parking owners but by a city (or whoever else)"_ the international user would be understandably **pretty pissed about that piece of misinformation**, as (s)he **doesn't care who collects the fee** nor how it is distributed afterwards nor how is fee named/defined, but only **whether fee exists or not** (for _their_ use case). 

Same issue is with ATMs. `fee=no` is only for situations where fee is never collected for anyone under any circumstances, eg. when somebody (and anybody!) withdraw say `50 EUR` (or whatever) from ATM, their account balance only reduces by `50 EUR` (and not some arbitrary higher amount). 

If fee is sometimes charged, then it is `fee=some` (or even more elaborately defined, as shown in example in my [previous comment](https://github.com/streetcomplete/StreetComplete/issues/3100#issuecomment-897232648)).

Also consider that UK has less residents than tourists per year (see previous discussions), so it is **not** some _"minor and irrelevant misinformation which affects mostly no-one"._
